scripts/actions/send-and-update-translation-queue.js

```diff
@@ -201,79 +224,60 @@ const sendContentToVendor = async (content, accessToken) => {
   });
 
   const fileResponses = await Promise.all(fileRequests);
-  const numSuccess = fileResponses.filter(({ code }) => code === 'ACCEPTED');
-
+  const successfulUploads = fileResponses.filter(
+    ({ code }) => code === 'ACCEPTED'
+  );
+  await updateStatus(successfulUploads, 'IN_PROGRESS');
   console.log(
-    `[*] Successfully uploaded ${numSuccess.length} / ${fileResponses.length} files`
+    `[*] Successfully uploaded ${successfulUploads.length} / ${fileResponses.length} files`
   );
 
-  return { batchUids, fileResponses };
+  return { fileResponses };
 };
 
 /**
  * @param {string[]} batchUids A list of vendor UIDs to be added to the `being_translated` queue.
+ * @param {string} status Status of a job
  */
-const addToBeingTranslatedQueue = async (batchUids) => {
-  const table = 'TranslationQueues';
-  const key = { type: 'being_translated' };
-
-  const data = await loadFromDB(table, key);
-
-  // If this field is empty/returns as empty object
-  const queue =
-    data.Item && data.Item.batchUids && data.Item.batchUids.length
-      ? data.Item.batchUids
-      : [];
-
-  await saveToTranslationQueue(key, 'set batchUids = :batchUids', {
-    ':batchUids': [...queue, ...batchUids],
-  });
+const updateStatus = async (successfulUploads, status) => {
+  for (const upload of successfulUploads) {
+    const batch_uid = upload.batchUid;
+    const job = await getJobs({ batch_uid });
+    const translationId = await getTranslationsJobsRecords({ jobId: job.id });
+    await updateJob(job.id, { status });
+    await updateTranslation(translationId, { status });
+  }
 };
 
 /**
- * Saves any files that failed to upload to the "to be translated" queue.
- * @param {{code: string, locale: string, slug: string}[]} failedUploads
- * @returns {Promise<boolean>}
+ * @param {Object[]} failedUploads A list of failed requests to the vendor
  */
-const saveFailedUploads = async (failedUploads) => {
-  const updatedLocales = failedUploads.reduce(
-    (acc, page) => ({
-      ...acc,
-      [page.locale]: [...(acc[page.locale] || []), page.slug],
-    }),
-    {}
-  );
-
-  await saveToTranslationQueue(
-    { type: 'to_translate' },
-    'set locales = :locales',
-    { ':locales': updatedLocales }
-  );
+const outputFailedUploads = async (failedUploads) => {
+  for (const failedUpload of failedUploads) {
+    console.log(`    [!] ${JSON.stringify(failedUpload)}`);
+  }
```

Comment:
i think we will also want to mark this as a failure in the workflow in addition to printing out the failed files.